filer/admin/imageadmin.py

```diff
@@ -90,11 +90,17 @@ class Media(object):
 
 
 class ImageAdmin(FileAdmin):
+    change_form_template = 'admin/filer/image/change_form.html'
     form = ImageAdminForm
 
 
+if FILER_IMAGE_MODEL == 'filer.Image':
+    extra_main_fields = ('author', 'default_alt_text', 'default_caption',),
```

Comment:
looks like there's an extra comma at the end

Reply:
Fixed